Doc/library/imaplib.rst

```diff
@@ -53,7 +53,7 @@ base class:
       Support for the :keyword:`with` statement was added.
 
    .. versionchanged:: 3.9
-      The optional *timeout* parameter was added.
+      The optional *timeout* parameter was added and :meth:`IMAP4.unselect` was added.
```

Comment:
I think that for `unselect` this should be `versionadded`, and so not combined with the `timeout` `versionchanged`.

See https://devguide.python.org/documenting/#paragraph-level-markup. There are also various discussions of this on bpo, such as https://bugs.python.org/issue26366. Although I admit this is a gray area, and not super important.